Source/BugsnagCrashReport.m

```diff
@@ -309,6 +310,19 @@ - (instancetype)initWithKSReport:(NSDictionary *)report {
       _groupingHash = BSGParseGroupingHash(report, _metaData);
       _overrides = [report valueForKeyPath:@"user.overrides"];
       _customException = BSGParseCustomException(report, [_errorClass copy], [_errorMessage copy]);
+      
+      
+      NSDictionary *recordedState = [report valueForKeyPath:@"user.eventHandledState"];
+      
+      if (recordedState) {
+          _eventHandledState = recordedState;
+      } else { // the event was unhandled.
+          _eventHandledState = @{
+                                 @"unhandled": @YES,
+                                 @"originalSeverity": BSGFormatSeverity(BSGSeverityWarning),
```

Comment:
Should this be `error`? I'm not sure when this case would happen.

Reply:
Yes - I've updated this so that unhandled events default to a severity of Error.